README.md

```diff
@@ -33,70 +33,72 @@ We will define a `set` and fill it up with the tokens which current query should
 key_tokens = set()
 ```
 
-Nearby tokens are important obviously because in a sentence (sequence of words), current word is highly dependent on few future & past tokens. This idea introduced the concept of `sliding attention`.
+Nearby tokens are important because in a sentence (sequence of words), the current word is highly dependent on neighboring future & past tokens. This intuition is the idea behind the concept of `sliding attention`.
 
 ```python
 # Let's update `set` with nearby tokens
 sliding_tokens = ["<something>", "....."]
 key_tokens.update(sliding_tokens)
```

Comment:
I have a hard time undrestanding this

Reply:
I would probably define one input sentence of let's say 10 words and then say let's see what tokens the 3rd word attends to.

E.g. in "Today is a nice day and I will go hiking"

we collect all other words "nice" is attending to assuming that "Today is a global token and the sliding window length is 2 

then you would collect "Today" (for global attn), "nice" and "day" (for sliding) and let's say "will" for random. I think this would make everything a bit clearer than always adding "<something>"